### PR TITLE
Docker build and a first usage to close #109

### LINKED ISSUE #109: Automatically add RSVP widget for meetups
It'd be great if we could parse the meetup.com event id for a meetup via a tag in the event description and automatically generate RSVP widgets for all meetups on our calendar section.

Ideally we want the RSVP button, right below the time/date div and horizontally aligned in relation to that.

Generate mock widgets here: http://www.meetup.com/meetup_api/buttons/

### PR DESCRIPTION
**Pull request by theodorosploumis**
- Add member to people
 - Add meetup.com rsvp support. See #109 
 - Add Docker instructions on README

Good luck!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,67 @@ SKGTech is using a [Public Google Calendar](https://www.google.com/calendar/embe
...
+
+If you are tired of installing npm, gem and other stuff there is a [Docker](http://docker.com) [image](https://hub.docker.com/r/skgtech/website/) for you.
+
+See more at [github.com/skgtech/skgtech.io-docker](https://github.com/skgtech/skgtech.io-docker/).
+
+Here is an example of steps of using this Docker image.
+
+Pull the latest docker image
+```
+git pull skgtech/website
```
Comment (kbariotis) [inline README.md]: Shouldn't this be `docker`?

Comment (theodorosploumis) [inline README.md]: The info are to use Docker to build the SKGTech.io website. Not to build any docker image etc. 
Did not understand well?

Comment (kbariotis) [inline README.md]: Should the `git pull skgtech/website` be `docker pull skgtech/website`?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,67 @@ SKGTech is using a [Public Google Calendar](https://www.google.com/calendar/embe
...
+
+If you are tired of installing npm, gem and other stuff there is a [Docker](http://docker.com) [image](https://hub.docker.com/r/skgtech/website/) for you.
+
+See more at [github.com/skgtech/skgtech.io-docker](https://github.com/skgtech/skgtech.io-docker/).
+
+Here is an example of steps of using this Docker image.
+
+Pull the latest docker image
+```
+docker pull skgtech/website
```
Comment (theodorosploumis) [inline README.md]: Ready.

**Comment by theodorosploumis** [comment]
Shall we merge it?

**Comment by kbariotis** [comment]
@theodorosploumis didn't find the time to test it. Go ahead if you're sure that its working. :)

**Comment by theodorosploumis** [comment]
Merged it. How is the live site rebuild? I can still see the same code on
http://skgtech.io.

**Comment by sirodoht** [comment]
@theodorosploumis Github cannot build our front JS app, since it does not have npm and browserify. It only build Jekyll (and Sass which is included on the Jekyll build).

So, we need to build it ourselves using `gulp scripts` -- maybe it's not very clear the readme.
See #129

**Comment by theodorosploumis** [comment]
You have been invited to the following event.

Title: Introduction to User Experience and Usability
Venue: ApoDec
Info:  
https://www.eventbrite.com/e/introduction-to-user-experience-and-usability-tickets-35436700124
Map: https://goo.gl/maps/UnsAEKFsHwC2
About: Here comes the very 1st UX event in Thessaloniki organized by  
ApoDec.I nvited guest speaker is Dr. Panagiotis Zaharias who is a  
well-known expert in the field with more than 17 years of experience both  
in academic and business settings. Panagiotis is also the organizer of the  
first UX meetup in Greece, called "Athens UX Community".

When: Tue Jul 4, 2017 19:00 – 21:00 Athens
Where: Str. Sfetsou 3, Thessaloniki 546 26, Greece
Calendar:  
reply@reply.github.com
Who:
     * Theodoros Ploumis - creator
     *  
reply@reply.github.com

Event details:  
https://www.google.com/calendar/event?action=VIEW&eid=N3E3azdzbDduZjR2OG83M2ppNDdxdXJqY2cgcmVwbHkrMDAxNDExZjlkMzUyZjk5NzViY2ZiNjAzYjQ0MzIyN2QwNmE0Nzg4ZGU5Nzk2YmVkOTJjZjAwMDAwMDAxMTMxZTI1MmU5MmExNjljZTA4YjgxYjg1QHJlcGx5LmdpdGh1Yi5jb20&tok=NTIjaHBtOTNnN2pmOGFnOTUwc212Y21sdjN1bTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbWE4NTI4ZTQ2NzI4ODViMzM1ZWFhZThiNjA3M2Y3NGQ2MGViY2U0OGE&ctz=Europe/Athens&hl=en

Invitation from Google Calendar: https://www.google.com/calendar/

You are receiving this courtesy email at the account  
reply@reply.github.com  
because you are an attendee of this event.

To stop receiving future updates for this event, decline this event.  
Alternatively you can sign up for a Google account at  
https://www.google.com/calendar/ and control your notification settings for  
your entire calendar.

Forwarding this invitation could allow any recipient to modify your RSVP  
response. Learn more at  
https://support.google.com/calendar/answer/37135#forwarding

**Comment by theodorosploumis** [comment]
This event has been canceled and removed from your calendar.

Title: Introduction to User Experience and Usability
Venue: ApoDec
Info:  
https://www.eventbrite.com/e/introduction-to-user-experience-and-usability-tickets-35436700124
Map: https://goo.gl/maps/UnsAEKFsHwC2
About: Here comes the very 1st UX event in Thessaloniki organized by  
ApoDec.I nvited guest speaker is Dr. Panagiotis Zaharias who is a  
well-known expert in the field with more than 17 years of experience both  
in academic and business settings. Panagiotis is also the organizer of the  
first UX meetup in Greece, called "Athens UX Community".

When: Tue Jul 4, 2017 19:00 – 21:00 Athens
Where: Str. Sfetsou 3, Thessaloniki 546 26, Greece
Calendar:  
reply@reply.github.com
Who:
     * Theodoros Ploumis - creator


Invitation from Google Calendar: https://www.google.com/calendar/

You are receiving this courtesy email at the account  
reply@reply.github.com  
because you are an attendee of this event.

To stop receiving future updates for this event, decline this event.  
Alternatively you can sign up for a Google account at  
https://www.google.com/calendar/ and control your notification settings for  
your entire calendar.

Forwarding this invitation could allow any recipient to modify your RSVP  
response. Learn more at  
https://support.google.com/calendar/answer/37135#forwarding

**Pull request merged**